packages/truffle/package.json

```diff
@@ -4,7 +4,10 @@
   "description": "Truffle - Simple development framework for Ethereum",
   "author": "consensys.net",
   "homepage": "https://github.com/trufflesuite/truffle/",
-  "repository": "https://github.com/trufflesuite/truffle/tree/master/packages/truffle",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/trufflesuite/truffle.git"
```

Comment:
Does this one not need a `directory`?

Reply:
Hm, I guess it should have one.  I'll go change that.